feature/container/containerz/tests/container_lifecycle/README.md

```diff
@@ -98,6 +98,23 @@ Using the container started as part of CNTR-1.2, validate that the container can
 be stopped, and is subsequently no longer listed in the `gnoi.Containerz.List`
 API.
 
+## CNTR-1.5: Restart Behaviour
+
+Using the containers started as part of CNTR-1.1, restart the device via gNOI
+and ensure that the containers are still running post reboot.
+
+## CNTR-1.6: Failover Behaviour
+
+Using the containers started as part of CNTR-1.1, trigger a failover and 
```

Comment:
trigger what kind of failover?   Does this mean this now needs to run on a MFF device (with redundant CONTROLLER_CARD components?). If yes, please also add

gnoi.system.System.SwitchControlProcessor to the rpc's below.

Please also add a heading at the bottom of the file for the requirement DUT platform for MFF.
(ref: https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#required-dut-platform)

Reply:
done